specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/gitintegration.json

```diff
@@ -0,0 +1,123 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "ArtifactsClient"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/getGitHubAccessToken": {
+      "post": {
+        "tags": [
+          "WorkspaceGitRepoManagement_getGitHubAccessToken"
+        ],
+        "operationId": "Get the GitHub access token.",
```

Comment:
>"operationId": "Get the GitHub access token.", [](start = 8, length = 46)

Sorry I didn't make this clear, the operation id should be `WorkspaceGitRepoManagement_getGitHubAccessToken`